packages/react/src/components/progress/progress.tsx

```diff
@@ -0,0 +1,38 @@
+import { c, classy, m, ProgressProps as BaseProps } from '@onfido/castor';
+import React from 'react';
+
+export const Progress = ({
+  value = 0,
+  max = 100,
+  size = 'regular',
+  hideLabel,
+  children,
+  className,
+}: ProgressProps): JSX.Element => {
+  const normalizedValue = Math.round((value / max) * 100);
+
+  return (
+    <div
+      className={classy(c('progress'), m(size), className)}
+      role="progressbar"
+      aria-valuenow={value}
+      aria-valuemin={0}
+      aria-valuemax={max}
+      aria-valuetext={!hideLabel && children ? children : undefined}
+    >
+      <div className={classy(c('progress-background'), m(size))}>
+        <div
+          className={classy(c('progress-indicator'))}
+          style={{ width: `${normalizedValue}%` }}
+        />
+      </div>
+      {!hideLabel && (
+        <div className={classy(c('progress-label'), m(size))}>
+          {children || `${normalizedValue}%`}
+        </div>
+      )}
+    </div>
+  );
+};
+
+export type ProgressProps = BaseProps & JSX.IntrinsicElements['progress'];
```

Comment:
The element used is actually not a `<progress>` but a `<div>`. That's why props of a JSX should also be of that component:

```suggestion
export type ProgressProps = BaseProps & JSX.IntrinsicElements['div'];
```

Then, I would exclude all the types that are overridden in the component. When setting such on TypeScript you'd get an error, which is better then putting something in but getting it rewritten.

```suggestion
export type ProgressProps = BaseProps & Omit<JSX.IntrinsicElements['div'], 'aria-valuemax' | 'aria-valuemin' | 'aria-valuenow' | 'aria-valuetext' | 'role'>;
```

P.S. The last one will fail, as it will need to be run against Prettier if you accept the suggestion.

Reply:
Done!